plot-api/src/main/kotlin/jetbrains/letsPlot/theme.kt

```diff
@@ -47,6 +47,10 @@ class theme() : OptionsMap(Option.Plot.THEME, HashMap()) {
     fun legendPosition_right() = withOption(LEGEND_POSITION, VAL_LEGEND_POS_RIGHT)
     fun legendPosition_bottom() = withOption(LEGEND_POSITION, VAL_LEGEND_POS_BOTTOM)
     fun legendPosition_top() = withOption(LEGEND_POSITION, VAL_LEGEND_POS_TOP)
+    fun tooltip_anchor_top_right() = withOption(TOOLTIP_ANCHOR, VAL_TOOLTIP_ANCHOR_TOP_RIGHT)
```

Comment:
Let's use signature 'tooltipAnchor_topRight()'

Reply:
I'm going to remove these methods from Theme to be later added to layer_tooltips()